cla_backend/apps/knowledgebase/tests/admin/test_command.py

```diff
@@ -0,0 +1,25 @@
+from django.test import TestCase
+from django.contrib.auth.models import Permission, Group, ContentType
+from django.core.management import call_command
+from django.contrib.auth.models import User
+from django.shortcuts import get_object_or_404
+from core.tests.mommy_utils import make_user
+
+
+class ArticleCategoryClaSuperUsersPermissionCommandTest(TestCase):
+    def test_command(self):
+        group = Group.objects.get(name="CLA Superusers")
+        cla_superuser = make_user()
+        group.user_set.add(cla_superuser)
+
+        content_type = ContentType.objects.get(app_label="knowledgebase", model="articlecategory")
+        perms = [
+            "knowledgebase.{}".format(perm.codename) for perm in Permission.objects.filter(content_type=content_type)
+        ]
+        self.assertFalse(cla_superuser.has_perms(perms))
+
+        call_command("cla_superusers_article_categories_permissions")
+
+        # Reload the user object
+        user = get_object_or_404(User, pk=cla_superuser.pk)
```

Comment:
Seems weird to me to use `get_object_or_404()` here, rather than just `User.objectsget()`, with a try/except if you really need it